### ISSUE TITLE
Version issues

### ISSUE THREAD

**Issue by hbaker2** [tyiannak/amvoc#52]
Hello, I am having some issues with the current versions of these functions, is there an updated code I could use please? Thank you.

**Comment by NickleDave** [comment]
Hi @hbaker2 I'm not one of the developers, but it might help if you add more detail about what issues you're running into.
Can you provide a full traceback of the errors you're getting, and include the versions of packages installed into your environment? For example see the bug report template here: https://github.com/vocalpy/vak/issues/new?assignees=&labels=BUG&projects=&template=bug_report.md&title=BUG%3A

**Comment by hbaker2** [comment]
Sure, thank you for taking the time to help me, I am looking forward to utilizing this package. 
Basically, I first changed my directory in Anaconda prompt to the amvoc-master folder which I had downloaded and relocated to my Desktop.
Then, I followed the instructions by running the code for
`pip3 install -r requirements.txt`
Then I changed directory to the src folder inside of amvoc-master in which the main.py file is located and ran the code for
`python3 main.py -i data/B148_test_small.wav -c n`
And the message I get is:
```
UserWarning: The dash_table package is deprecated. Please replace
`import dash_table` with `from dash import dash_table`
```
I also get an error for:
`ModuleNotFoundError: No module named 'pyAudioAnalysis'`
I get these errors even though I confirmed that both of the packages are already installed. 
Thank you for your help.
